tpl/template_embedded.go

```diff
@@ -233,7 +233,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
 {{ end }}{{ end }}
 
 <!-- Output all taxonomies as schema.org keywords -->
-<meta itemprop="keywords" content="{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}" />
+<meta itemprop="keywords" content="{{ if .IsPage}}{{ range $index, $tag := .Params.tags }}{{ $tag }},{{ end }}{{ else }}{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}{{ end }}" />
```

Comment:
I have two concerns to share:

First, we could be using `delimit` to clean it up:

```html
<meta name="keywords" content="{{ if .IsPage }}{{ delimit .Params.tags ", " }}{{ end }}">
```

Second, this assumes the schema template is only being used once in a document. However, on list pages it's quite likely schema will need to be output multiple times. As a result, we should not duplicate the `.Site.Taxonomies` information using an accumulator as the output in a list would be duplicative.